packages/telemetry-node/package.json

```
@@ -0,0 +1,35 @@
+{
+  "name": "@replayio/telemetry-node",
+  "version": "1.0.0",
+  "description": "Node utilities for telemetry",
+  "main": "./dist/index.js",
+  "exports": {
+    ".": "./dist/index.js",
+    "./package.json": "./package.json",
+    "./telemetry": "./dist/telemetry.js"
```

this `/telemetry` entrypoint is redundant, the main entrypoint already exports everything from it
```suggestion
    ".": "./dist/index.js",
    "./package.json": "./package.json"
```